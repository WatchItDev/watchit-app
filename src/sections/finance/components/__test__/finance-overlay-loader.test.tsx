import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";
import FinanceOverlayLoader from "../finance-overlay-loader";

describe("[COMPONENTS] <FinanceOverlayLoader/>", () => {
  it("to match snapshot", () => {
    const { container } = render(<FinanceOverlayLoader />);
    expect(container).toMatchSnapshot();
  });
});
