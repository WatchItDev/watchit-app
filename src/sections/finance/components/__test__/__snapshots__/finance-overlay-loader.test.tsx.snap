// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`[COMPONENTS] <FinanceOverlayLoader/> > to match snapshot 1`] = `
<div>
  <div
    class="MuiBox-root css-4c6let"
  >
    <div
      class="MuiBox-root css-tk1iyj"
    >
      <span
        class="MuiLinearProgress-root MuiLinearProgress-colorInherit MuiLinearProgress-indeterminate css-yptbrg-MuiLinearProgress-root"
        role="progressbar"
      >
        <span
          class="MuiLinearProgress-bar MuiLinearProgress-barColorInherit MuiLinearProgress-bar1Indeterminate css-jlm0hp-MuiLinearProgress-bar1"
        />
        <span
          class="MuiLinearProgress-bar MuiLinearProgress-barColorInherit MuiLinearProgress-bar2Indeterminate css-1wsh2fw-MuiLinearProgress-bar2"
        />
      </span>
    </div>
  </div>
</div>
`;
